### DISCUSSION TITLE
First Init Sign client, Pairing don't trigger session_proposal

### DISCUSSION THREAD

**Discussion by Hiraki99** [WalletConnect/walletconnect-monorepo#3969] [Developer Support] [answered]
I'm using sign api of walletconnect v2. First install app, I init walletconnect and scan qr code to pairing to D-app.
I pairing app success, but i wait very long that dont receive data on session_proposal event. I quit my app and reconnect, everything work very well.
How can I fix this error on first init my app.
I'm using RN to build my wallet.

**Comment by Hiraki99** [comment]
@ignaciosantise help me

**Comment by glitch-txs** [comment]
still need help? could you share your code?

**Reply by Hiraki99** [reply, marked as answer]
I fixed this error, after pairing, i'm open transport on relayer, full flow sign client worked
